### PR TITLE
Remove share/genbuild.sh and its (stale) uses

### PR DESCRIPTION
**Pull request by daira**
Signed-off-by: Daira Hopwood <daira@jacaranda.org>

### CODE REVIEW

**Comment by daira** [comment]
@zkbot try

File: Makefile.am [hunk 1]
```diff
@@ -149,7 +149,7 @@ endif
 dist_bin_SCRIPTS = zcutil/fetch-params.sh
 dist_noinst_SCRIPTS = autogen.sh zcutil/build-debian-package.sh zcutil/build.sh
 
-EXTRA_DIST = $(top_srcdir)/share/genbuild.sh qa/pull-tester/rpc-tests.sh qa/pull-tester/run-bitcoin-cli qa/rpc-tests qa/zcash $(DIST_DOCS) $(BIN_CHECKS)
```
Comment (etardz) [inline Makefile.am]: Duplicate of #

**Comment by daira** [comment]
etardz is a spam account and its review should not be counted.

**Comment by daira** [comment]
`share/genbuild.sh` is used to generate `src/obj/build.h`. I was mistaken in thinking that the latter is not used (it's `#include`d by `src/clientversion.cpp`).

**Pull request closed without merging**